axelor-prestashop/src/main/java/com/axelor/apps/prestashop/web/AppPrestaShopController.java

```diff
@@ -57,15 +57,20 @@ public void testConnection(ActionRequest request, ActionResponse response)
     final List<String> errors = new LinkedList<>();
     final List<String> warnings = new LinkedList<>();
     final List<String> info = new LinkedList<>();
+
+    response.setValue("isValid", false);
+
     service.checkAccess(appConfig, errors, warnings, info);
 
     if (errors.isEmpty() == false) {
```

Comment:
correct all such code where == false/true is compared, no needs to compare boolean values with true/false, we can directly use it